www/main/settings/components/SettingViewer.jsx

```diff
@@ -1,39 +1,22 @@
-import React, { useEffect, useState } from 'react';
+import React from 'react';
 import { useStoreState } from 'easy-peasy';
 
 const themes = require('../../../configs/themes.json');
 
 const SettingViewer = () => {
   const { themeBg } = useStoreState((state) => state.userSettings);
-  const { slideOrder } = useStoreState((state) => state.viewerSettings);
-
-  const [translationOrder, setTranslationOrder] = useState();
-  const [teekaOrder, setTeekaOrder] = useState();
-  const [transliterationOrder, setTransliterationOrder] = useState();
-
-  const orderFunctions = {
-    translation: (item) => {
-      if (translationOrder !== item) {
-        setTranslationOrder(item);
-      }
-    },
-    transliteration: (item) => {
-      if (transliterationOrder !== item) {
-        setTransliterationOrder(item);
-      }
-    },
-    teeka: (item) => {
-      if (teekaOrder !== item) {
-        setTeekaOrder(item);
-      }
-    },
-  };
 
   const {
     gurbaniFontSize,
     content1FontSize,
     content2FontSize,
     content3FontSize,
+    content1Visibility,
+    content2Visibility,
+    content3Visibility,
+    content1,
+    content2,
+    content3,
```

Comment:
Veerji, what do these content variables represent? 

PS I think this should be renamed to a self-explanatory name

Reply:
Initially we had translation, transliteration and teeka. Now instead of that, we are using content1, content2, and content3, because that can be switched.

Context: https://github.com/KhalisFoundation/sttm-desktop/pull/1903

Reply:
Veerji I still doubt the naming convention. It would always need an explanation to it's name. We have to come up with better specific name. 

Maybe something like `contentLine1`, `contentLine2`, or `contentLine3` can be a better name. This is just a rough idea to target what exact content we are referring to specifically.

Reply:
Moving this to Code refactors issue #1802